core/Cargo.toml

```diff
@@ -21,7 +21,7 @@ std = ["math/std", "winter-utils/std"]
 
 [dependencies]
 math = { package = "winter-math", version = "0.6", default-features = false }
-crypto = { package = "miden-crypto", version = "0.6", default-features = false }
+crypto = { package = "miden-crypto", git = "https://github.com/0xPolygonMiden/crypto.git", branch = "frisitano-tx-result", default-features = false }
```

Comment:
I might have missed it - but seems like the changes in this PR should work fine with `v0.6`. If so, let's revert back here.

Reply:
That's correct however I needed to have consistent versions of crypto for my work in `miden-base`. It's not a blocker for this PR.  I will revert v0.6, merge and then restore the branch and continue using it for downstream (miden-base) dev.